src/main/java/de/adesso/kicker/notification/NotificationController.java

```diff
@@ -2,43 +2,70 @@
 
 import java.util.List;
 
+import de.adesso.kicker.notification.teamjoinrequest.TeamJoinRequestService;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 import org.springframework.web.servlet.ModelAndView;
 
-import de.adesso.kicker.team.TeamService;
-import de.adesso.kicker.user.UserService;
-
 @RestController
 public class NotificationController {
```

Comment:
Is this controller necessary? Notifications are mostly to be handled in the backend.

Reply:
Yes it is because the accept and decline calls are made by JS.